buildout-mercator.cfg

```diff
@@ -9,6 +9,7 @@ dump-requirements-file = ${make_wheels:platform}-requirements.txt
 frontend.static_dir = src/mercator/mercator/build
 frontend_package_name = mercator
 backend_package_name = adhocracy_mercator
+frontend.profile_images_enabled = false
```

Comment:
I would like to disable this for all liqd projects for now. This will make it fully backwards compatible and allow us to activate it as we see fit. @nidico can you please take care of this?